service/controller/v17/key/key.go

```diff
@@ -223,6 +227,21 @@ func IsPodDrained(pod *corev1.Pod) (bool, error) {
 	return b, nil
 }
 
+func KubeletVolumeSizeFromNode(node v1alpha1.KVMConfigSpecKVMNode) string {
+	// TODO: calvix
```

Comment:
```suggestion
	// TODO: https://github.com/giantswarm/giantswarm/issues/4105#issuecomment-421772917
```
To be explicit, add the TODO issue link here.